src/ProcessedStory.ts

```diff
@@ -3,10 +3,11 @@ import type { StorybookStory } from './browser/StorybookPage';
 
 type Params = {
   skip: boolean;
-  disabledRules: string[],
+  disabledRules: string[];
+  timeout: number;
   /** @deprecated */
-  waitForSelector?: string,
-}
+  waitForSelector?: string;
+};
```

Comment:
Right now this repo doesn't provide its own typing. When this PR merges, we should also update https://github.com/FB-PLP/traject/blob/master/typings/storybook-addon.d.ts#L6-L9 so LP has valid types

Reply:
Yeah, that'd work 😆